src/databricks/labs/ucx/contexts/application.py

```diff
@@ -316,9 +320,15 @@ def acl_migrator(self):
             self.config.inventory_database,
         )
 
+    @cached_property
+    def table_ownership_grant_loader(self) -> TableOwnershipGrantLoader:
+        return TableOwnershipGrantLoader(self.tables_crawler, self.table_ownership)
+
     @cached_property
     def migrate_grants(self) -> MigrateGrants:
+        # owner grants have to come first
```

Comment:
Why?

Reply:
because of `match_grants` returning the first matched grant

Reply:
Please add that to the comment